conf.d/health_alarm_notify.conf

```diff
@@ -61,6 +61,22 @@ curl=""
 # If not found, irc notifications will be silently disabled.
 nc=""
 
+#------------------------------------------------------------------------------
+# extra options for external commands
+#
+# In some cases, you may need to change what options get passed to an
+# external command.  Such cases are covered here.
+
+# Control TLS certificate validation for cURL.
+# If set to yes, disable certificatie validation.
+# Otherwise, cURL will validate TLS certificates normally.
+# ******************************WARNING****************************************
+# Disabling this validation by setting the below variable to 'yes'
+# makes it trivial for someone to block notification delivery through most
+# mechanisms other than email and IRC, and may additionally leak sensitive
+# information about the system state.
+curl_insecure="no"
```

Comment:
I suggest to make this:

```
# curl_options="--insecure"
```

That is:

1. commented
2. ready to be uncommented

Also, the comment above about "blocking notification delivery" when certificate verification is disabled, I believe is wrong. Only the "leak" is valid.

Reply:
Also please adapt the script accordingly for this change.